### ISSUE TITLE
break when save model

### ISSUE THREAD

**Issue by jjkke88** [zimenglan-sysu-512/Focal-Loss#3]
when I save model, the script crash , the usage of focalloss in my code is below
```
net.loss = L.FocalLoss(net.deconv, net.label, loss_weight=1, propagate_down=[1, 0], loss_param=dict(normalize=True, ignore_label=-1), focal_loss_param=dict(alpha=0.25, gamma=2))
````

**Comment by zimenglan-sysu-512** [comment]
hi @jjkke88,

which framework do you use?

**Comment by jjkke88** [comment]
caffe
